mango-viz/bdgenomics/io/utils.py

```diff
@@ -0,0 +1,72 @@
+import json
+import pandas
+
```

Comment:
- In https://github.com/bigdatagenomics/mango/blob/master/mango-viz/requirements.txt put pandas and modin as requirements
- required functions for each file type (bed, vcf, etc): `read` (seen by the user), `_parseDataframe` (internal),`_toJson` (internal)


Usage:
```
df = bdgenomics.mango.io.read_bed(file, use_pandas = False) # use_pandas will be set by user
tr = Track(source = DataframeSource(df), ...)
```